packages/@sparrow-workspaces/src/features/workspace-actions/layout/WorkspaceActions.svelte

```diff
@@ -716,6 +722,27 @@
         />
       </div>
     </div>
+
+    <!-- Launch sparrow desktop -->
+    <div class="px-3 pb-3">
```

Comment:
@aakashreddy-p also add a condition that this button should be visible only in webapp not in desktop app

Reply:
@Astitva877 okay I will add the condition in it.

Reply:
@Astitva877 I will fix it